modules/redis-cluster/metadata.yaml

```diff
@@ -82,6 +82,12 @@ spec:
         description: List of consumer network where the network address of the discovery endpoint will be reserved, in the form of projects/{network_project_id_or_number}/global/networks/{network_id}. Currently, only one item is supported
         varType: list(string)
         required: true
+        connections:
+          - source:
+              source: github.com/terraform-google-modules/terraform-google-network//modules/vpc
+              version: ">= 9.1.0"
+            spec:
+              outputExpr: projects/${project_id}/global/networks/${network_id}
```

Comment:
network_id is already in format of `projects/{{project}}/global/networks/{{name}}` 
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_network#id-1